src/types/errors-enum.ts

```diff
@@ -0,0 +1,6 @@
+export enum ErrorType {
+  emptyInput = 'Title should not be empty', // писать с большой буквы
```

Comment:
```suggestion
  EmptyInput = 'Title should not be empty', // писать с большой буквы
```
Yes, in enum write properties in uppercase